storagedrive.py

```diff
@@ -0,0 +1,17 @@
+from abc import ABC, abstractmethod
+
+class StorageDrive(ABC):
+
+    @abstractmethod
+    def upload(self):
```

Comment:
`upload` function should define an argument such as filename etc, which should be the basis for input for it's derived classes.

Reply:
Similarly for `download` and `search` function.